docs/the-top-10/introduction_for_ot.md

```diff
@@ -0,0 +1,29 @@
+# The security World for OT: Why is it problematic?
+
+Malicious external actors were previously not a problem for the operators.  These dangers were not relevant. In addition, direct physical access was usually required to manipulate the system.
+
+OT operators are familiar with the production process. Cybersecurity is a new field in which there is little knowledge.
+
+## Problems with security
+
+Many cyber security measures make it difficult to analyze errors if there is a problem with the machine or system. For example, an encrypted transmission of data cannot easily be analyzed to determine whether the data is correct. 
```

Comment:
maybe add: On the other hand, if transmission occurs without integrity control, the receiver can neither depend upon knowing from whom the message is originally from, nor if it has been tampered during transmission.